buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -1559,6 +1561,7 @@ class BeamModulePlugin implements Plugin<Project> {
       project.tasks.create(name: name, type: Test) {
         group = "Verification"
         description = "Validates the PortableRunner with JobServer ${config.jobServerDriver}"
+        systemProperties config.systemProperties
         systemProperty "beamTestPipelineOptions", JsonOutput.toJson(beamTestPipelineOptions)
```

Comment:
Shall we also merge this system property to systemProperties set above,

Reply:
updated